src/components/QuoteBoxCarousel/Item.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ProfileCard from '../ProfileCard'
+import QuoteBox from '../QuoteBox'
+
+interface CarouselItem {
+  readonly imagePath: string
+  readonly alternate: string
+  readonly name: string
+  readonly role: string
+  readonly quote: string
+}
+
+const CarouselItem = ({ imagePath, alternate, name, role, quote }: CarouselItem) => {
```

Comment:
Currently you are using arguments from this function to create `ProfileCard` and `QuoteBox`, what do you think of changing `CarouselItem` parameters to accept `ProfileCard` and `QuoteBox` instead?

Reply:
Very interesting 👏